packages/portal/spec-renderer/src/components/SpecDetails.cy.ts

```diff
@@ -0,0 +1,88 @@
+// Cypress component test spec file
+
+import { OperationListItem } from '../types'
+import { jsonSpec, specOpItem } from '../../fixtures/spec-data'
+import SpecDetails from './SpecDetails.vue'
+
+describe('<SpecDetails />', () => {
+  it('renders correctly with props', () => {
+    cy.mount(SpecDetails, {
+      props: {
+        document: jsonSpec as any,
```

Comment:
question: if you're going to cast the type, can't you cast it the same as the PropType in the component?